### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone. Use `ADD` instead.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/sdl-ttf-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name. Tests has been renamed to testsuite.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#include <stdint.h>
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Made the requested changes, thank you!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**